core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -211,6 +212,22 @@ public void hadoopShouldLoadFileSystemWhenConfigured() throws Exception {
     }
   }
 
+  /**
+   * Ensures that Hadoop loads the Alluxio file system with Zookeeper URI.
```

Comment:
comment is same as method name, can just remove it

Reply:
Removed, thanks